### ISSUE TITLE
about thereg_loss

### ISSUE THREAD

**Issue by yangzhikai** [ceshine/fast-neural-style#5]
what is   ” reg_loss ”   stand for...

**Comment by ceshine** [comment]
It's regularization loss. You should be able to see that by [reading the code](https://github.com/ceshine/fast-neural-style/blob/f1f0880871265a46d7668f7cf13ed15e4c7975ad/notebooks/01-image-style-transfer.py#L192):

```
reg_loss = REGULARIZATION * (
torch.sum(torch.abs(y[:, :, :, :-1] - y[:, :, :, 1:])) + 
torch.sum(torch.abs(y[:, :, :-1, :] - y[:, :, 1:, :])))
```
